docs/6.bridge/2.typescript.md

```diff
@@ -5,6 +5,19 @@
 - Remove `@nuxt/typescript-build`: Bridge enables same functionality
 - Remove `@nuxt/typescript-runtime` and `nuxt-ts`: Nuxt 2 has built-in runtime support
 
+### Set `bridge.typescript`
+
+```ts
+import { defineNuxtConfig } from '@nuxt/bridge'
+
+export default defineNuxtConfig({
+  bridge: {
+    typescript: true,
```

Comment:
is this not enabled by default?

Reply:
It is enabled by default.
The first step is setting `bridge: false`, so I believe needs some explanation.
https://nuxt.com/docs/bridge/overview#update-nuxtconfig